packages/cymem/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: cymem
```

Comment:
Please add some basic tests for packages that are not pure python packages: `cymem`, `murmurhash`, `preshed`, `srsly`, `thinc`, `blis`.

Reply:
I added a basic test to srsly and thinc, the rest are cython packages, where I am not sure how one would call them from pyodide.